docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+      gates_to_exclude = ["RH"]
+      pass
+
+.. _backend_methods:
+
+Backend Methods Usage
+=====================
+
+In order to use quantum hardware, the user must use the backend's authentication method to prove that they have the right credentials to send instruction to the hardware. The authentication method is specified by the hardware provider. Providing authentication credentials is the first step in using ``QuantumBackend``. Below is an example of using IBM's hardware through ``QiskitBackend``.
+
+``QuantumSimulator``\ s run in your own environment and therefore can be used without needing authentication credentials.
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend creation example
+  :end-before: # End QuantumBackend creation example
+
+General Backend Methods
+-----------------------
+
+After intializing the backend we can send the ``Circuit`` and number of repetitions to the backend and read the measurement. For more information on creating circuits to send to a backend, refer to the :ref:`Circuits Guide <circuits_guide>`. For a single circuit, that is executed as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuit
+  :end-before: # End QuantumBackend run and measure circuit
+
+If you want to run multiple circuits with different number of samples, the code needs to be modified as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuitset
+  :end-before: # End QuantumBackend run and measure circuitset
+
+``QuantumBackend`` can also provide the distribution of the outcome measurement. It can be extracted using the following approach:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumbackend measurement distribution
+  :end-before: # End Quantumbackend measurement distribution
+
+In all instances, ``QuantumBackend`` automatically adds the measurement operators to the circuit. Therefore, the user is not required to include it in the circuit.
+
+Simulator-specific methods
+--------------------------
+
+``Quantumsimulator`` can be used to extract the wave function and expectation value of a given circuit. Below is an example of extracting these values using a ``Quantumsimulator`` called ``CirqSimulator``:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumsimulator examples
+  :end-before: # End Quantumsimulator examples
+
+Zapata's backends
+=================
+
+SymbolicSimulator
+-----------------
+
+Zapata's ``SymbolicSimulator`` is a type of :ref:`quantum simulator <quantum_simulator>` that's built specifically to allow for the evaluation of circuits with :ref:`symbolic gates <symbolic_gates>`. Because there is extra overhead to allow for this broader type of evaluation, ``SymbolicSimulator`` is not the most performant simulator, but it can be useful for deeply understanding small circuits.
```

Comment:
I'd also add sth along the lines:

As it doesn't require any extra dependencies apart from `orquestra-quantum`, it can be also used for testing purposes.

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64